src/vite-env.d.ts

```
@@ -1 +1,8 @@
 /// <reference types="vite/client" />
+
+declare module '*.svg?react' {
+	import React = require('react');
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)